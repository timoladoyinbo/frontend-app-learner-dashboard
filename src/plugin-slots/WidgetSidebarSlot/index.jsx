import React from 'react';

import { PluginSlot } from '@openedx/frontend-plugin-framework';
import LookingForChallengeWidget from 'widgets/LookingForChallengeWidget';

// eslint-disable-next-line arrow-body-style
export const WidgetSidebarSlot = () => (
  <PluginSlot id="widget_sidebar_slot">
    <LookingForChallengeWidget />
  </PluginSlot>
);

export default WidgetSidebarSlot;
