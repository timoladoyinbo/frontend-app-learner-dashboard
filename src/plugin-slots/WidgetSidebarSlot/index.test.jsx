import { shallow } from '@edx/react-unit-test-utils';

import WidgetSidebarSlot from '.';

jest.mock('widgets/LookingForChallengeWidget', () => 'LookingForChallengeWidget');

jest.mock('@openedx/frontend-plugin-framework', () => ({
  PluginSlot: 'PluginSlot',
}));

describe('WidgetSidebar', () => {
  beforeEach(() => jest.resetAllMocks());

  test('snapshots', () => {
    const wrapper = shallow(<WidgetSidebarSlot />);
    expect(wrapper.snapshot).toMatchSnapshot();
  });
});
