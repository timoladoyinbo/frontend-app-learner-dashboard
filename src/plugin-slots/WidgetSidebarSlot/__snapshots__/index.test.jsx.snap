// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`WidgetSidebar snapshots 1`] = `
<PluginSlot
  id="widget_sidebar_slot"
>
  <LookingForChallengeWidget />
</PluginSlot>
`;
